config/ePO-default/config-parameters.xsl

```diff
@@ -20,7 +20,8 @@
             <xd:p>This module defines project level variables and parameters</xd:p>
         </xd:desc>
     </xd:doc>
-
+    <!-- List of xmi files to be merged into a single xmi file-->
+    <xsl:variable name="xmiFileList" select="collection('../../test/test-multi-xmi?select=*.xmi')"/>
```

Comment:
that is about right. Now We need to figure out with Andreea how will this be used in the ePO ontology.  Perhaps it will be a series of steps in a bash  script.